### PR TITLE
Added attention to online parameter changes 

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5274; https://github.com/pingcap/docs-cn/pull/6114
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dynamic-config.md [hunk 1]
```diff
@@ -66,18 +66,22 @@ When using the `set config` statement, you can modify the configuration of a sin
 
 - Modify the configuration of all TiKV instances:
 
-    {{< copyable "sql" >}}
+> **Note:**
+>
+> It is recommended to wrap variable names in quotation marks.
```
Comment (TomShawn) [inline dynamic-config.md:71]:
```suggestion
> It is recommended to wrap variable names in backticks.
```

File: dynamic-config.md [hunk 2]
```diff
@@ -275,7 +279,7 @@ The following example shows how to modify `slow-threshold` online by using the `
 {{< copyable "sql" >}}
 
 ```sql
-set tidb_slow_log_threshold = 200;
+set `tidb_slow_log_threshold` = 200;
```
Comment (morgo) [inline dynamic-config.md:282]: This is valid syntax, but for consistency when setting a sysvar the docs currently don't escape the values. It is a slightly different case than `set config`, in that sysvars won't use any special characters that need escaping.

Comment (TomShawn) [reply dynamic-config.md:282]: @rleungx PTAL

Comment (xiongjiwei) [reply dynamic-config.md:282]: > in that sysvars won't use any special characters that need escaping

agree with @morgo

Comment (TomShawn) [reply dynamic-config.md:282]:
```suggestion
set tidb_slow_log_threshold = 200;
```

Comment (Joyinqin) [reply dynamic-config.md]: Done in https://github.com/pingcap/docs-cn/pull/6114

**Pull request merged**